node/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "data-avail"
-version = "1.8.0"
+version = "1.8.1"
```

Comment:
Should we update the versioning scheme here as well? cc: @markopoloparadox @Leouarz

Reply:
Possibly. Let's discuss this the next time we see each other :)